activerecord/lib/active_record/connection_adapters/postgresql_adapter.rb

```diff
@@ -1182,7 +1191,6 @@ def add_pg_decoders
           coders.each { |coder| map.add_coder(coder) }
           @raw_connection.type_map_for_results = map
 
-          @type_map_for_results = PG::TypeMapByOid.new
```

Comment:
Is this change still required?

This method is called from `configure_connection`, which will be called multiple times over the instance lifetime (given reconnects). I think we want a new type map each time in that case -- but otherwise we probably need to at least clear it out.

Reply:
Nope. It's a left over I need to remove